lib/DDG/Spice/GoWatchIt.pm

```diff
@@ -30,9 +32,10 @@ spice wrap_jsonp_callback => 1;
 handle remainder_lc => sub {
 
   return unless $_; # Guard against "no answer"
-  $_ =~ s/$killwords//g; # remove killwords
-  return trim($_); # trim spaces and return
+  return if $_ =~ m/^$stopwords$/g; # Guard against stop words "apple watch"
```

Comment:
I think we should compare the full query against these as the order matters: "apple watch" vs "watch apple".

We expose the full query in the `$req` object.